blocks/store/selectors.js

```diff
@@ -61,3 +62,37 @@ export function getDefaultBlockName( state ) {
 export function getFallbackBlockName( state ) {
 	return state.fallbackBlockName;
 }
+
+/**
+ * Returns an array with the child blocks of a given block.
+ *
+ * @param {Object} state     Data state.
+ * @param {string} blockName Block type name.
+ *
+ * @return {Array} Array of child block names.
+ */
+export const getChildBlockNames = createSelector(
```

Comment:
This selector seems to be quite important so let’s add unit tests to avoid regressions in the future.

Reply:
Unit tests added 👍